entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md

```diff
@@ -781,3 +781,80 @@ This change was made so that the version of SQLite used on iOS consistent with o
 **Mitigations**
 
 To use the native SQLite version on iOS, configure `Microsoft.Data.Sqlite` to use a different `SQLitePCLRaw` bundle.
+## Char values are now stored as TEXT on SQLite
+
+[Tracking Issue #15020](https://github.com/aspnet/EntityFrameworkCore/issues/15020)
+
+This change was introduced in EF Core 3.0-preview 4.
```

Comment:
I thought this was a change on the Microsoft.Data.Sqlite provider. Am I wrong?

Reply:
I also changed a fallback in RevEng, and the literal generation in EF Core.

Reply:
Then I guess I was mostly wrong :smile: